Mathlib/Data/Fin/Basic.lean

```diff
@@ -300,11 +300,12 @@ section Add
 ### addition, numerals, and coercion from Nat
 -/
 
-@[simp]
-theorem val_one' (n : ℕ) [NeZero n] : ((1 : Fin n) : ℕ) = 1 % n :=
-  rfl
+@[simp] theorem val_ofNat [NeZero n] : (ofNat(m) : Fin n).val = m % n := rfl
```

Comment:
I think this already exists as `Fin.coe_ofNat_eq_mod`

Reply:
(which also raises that the RHS is incorrect)

Reply:
I'll let this build first just to be sure nothing is sensitive to the change.

Reply:
I think when I tried it before `simp` caused trouble, but maybe I just was fixing something else and didn't want to scope creep.

Reply:
I also have a vague recollection that this wasn't made `simp` deliberately, but I also don't remember a concrete reason. So I think we should go with this!